.vsts-pipelines/builds/ci-public.yml

```diff
@@ -10,10 +10,41 @@ pr:
     include:
     - '*'
 
+schedules:
+- cron: 0 9 * * 1
+  displayName: "Run CodeQL3000 weekly, Monday at 2:00 AM PDT"
+  branches:
+    include:
+    - main
+  always: true
+
+parameters:
+# Parameters below are ignored in public builds.
+#
+# Choose whether to run the CodeQL3000 tasks.
+# Manual builds align w/ official builds unless this parameter is true.
+- name: runCodeQL3000
+  default: false
+  displayName: Run CodeQL3000 tasks
+  type: boolean
+
+
 phases:
   - template: ../templates/default-build.yml
     parameters:
       agentOs: Windows
-      codeSign: true
-      ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-        buildArgs: '/p:OfficialBuild=true /p:Sign=true'
+      ${{ if or(eq(variables['Build.Reason'], 'Schedule'), and(eq(variables['System.TeamProject'], 'internal'), eq(variables['Build.Reason'], 'Manual'), eq(parameters.runCodeQL3000, 'true'))) }}:
```

Comment:
Change the condition so that a scheduled public run does **not** do `CodeQL3000` things